website-1.x/blog/2021-05-11-releasing-docusaurus-beta.md

```diff
@@ -0,0 +1,84 @@
+
+[![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/favorites.png)](https://docusaurus.io/showcase?tags=favorite)
+
+## Why was Docusaurus v2 in alpha for so long?
+
+The first alpha release [v2.0.0-alpha.0](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.0) was **2 years ago** 😅.
+
+It was definitively a very long and unusual alpha period.
+
+Docusaurus 2 has reached **feature parity** with Docusaurus 1 with **[i18n](https://docusaurus.io/blog/2021/03/09/releasing-docusaurus-i18n)**, and it has been **successfully adopted** by many Docusaurus sites (see [i18n showcase](https://docusaurus.io/showcase?tags=i18n)).
+
+We're now confident that the **core features** of Docusaurus 2 are **stable enough** to be in beta.
+
+## What are the goals of the beta?
+
+We will use the beta phase to **improve our theming system**.
+
+We want to make it:
+
+- **easier to implement a custom theme**, including for ourselves. We want to provide [multiple official themes](https://github.com/facebook/docusaurus/issues/3522) (including [Tailwind CSS](https://github.com/facebook/docusaurus/issues/2961)) for a long time.
+
+- **safer to extend an existing theme**: it can be painful to upgrade a **highly customized** Docusaurus site, as customizations can **conflict with internal changes**. We need to make the theme public API surface more explicit, and make it clear what is safe to customize.
+
+We will build a better **theming infrastructure** and refactor the classic theme to use it.
+
+If you don't customize your Docusaurus site much, you can consider Docusaurus stable.
+
+## What's new?
+
+In case you missed it, we recently shipped two major improvements:
+
+- [Auto-generated sidebars](https://docusaurus.io/docs/sidebar#sidebar-item-autogenerated): no need to maintain a `sidebars.js` file anymore!
+- [Webpack 5 / PostCSS 8](https://github.com/facebook/docusaurus/issues/4027): persistent caching significantly speeds up **rebuild time**!
+
+## What's next?
+
+We will continue to **fix bugs** and implement the **most wanted features**, some of them are:
+
+- [Sidebar category index pages](https://github.com/facebook/docusaurus/issues/2643)
+- [Proper CSS-in-JS support](https://github.com/facebook/docusaurus/issues/3236)
+- [Extend Docusaurus plugins, CMS integration](https://github.com/facebook/docusaurus/issues/4138)
+- [Fix trailing slashes and relative link issues](https://github.com/facebook/docusaurus/issues/3372)
+- [Better compatibility with CommonMark](https://github.com/facebook/docusaurus/issues/3018)
+- [Upgrade to MDX 2.0](https://github.com/facebook/docusaurus/issues/4029)
+
+## Conclusion
+
+This is an **exciting time for Docusaurus**.
+
+We read a lot of **positive feedbacks** about Docusaurus, and discover **new sites online** every single day.
+
+We hope that removing the **alpha label** will be reassuring for the community, and **increase adoption** even more.
```

Comment:
```suggestion
We are so excited for this release, and hope that by moving from alpha to beta the community is assured on quality and stability, which, in turn, continues to increase the adoption of Docusaurus. And, for those that have been on the fence from migrating an existing site to Docusaurus 2, it is a great time to upgrade so that you are running on the latest infrastructure when we deprecate Docusaurus 1 at the end of this beta period.
```